cmake/Macros.cmake

```diff
@@ -248,4 +248,41 @@ function(lug_download_thirdparty)
     file(WRITE "${LUG_THIRDPARTY_DIR}/version" "${THIRDPARTY_PLATFORM}_${THIRDPARTY_SHORT_HASH}")
     message(STATUS "Done!")
 
+    file(REMOVE ${DL_FILE})
```

Comment:
Here you shouldn't remove the downloaded file because it can be reused in a later build (https://github.com/Lugdunum3D/LugBench/pull/29/files#diff-031c6047a0f4c227a313b7b2f3b77219R212)